example/app/config.yml

```diff
@@ -300,6 +300,7 @@ kk-uat-deployment:
 mth:
   stackName: mth
   stackNameNoDash: Mth
+  useWorkflowLambdaVersions: false
```

Comment:
This change is unrelated to the work done in the PR, it is just there to make my deployments more reliable.

Reply:
@yjpa7145  Should we be concerned that you needed to do this?

Reply:
I do it when I have to make bunches of changes to a single lambda and want to deploy just that one.